scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -75,7 +75,7 @@ class PostgresStorage(
         val statement = connection.createStatement()
         val resultSet = statement.executeQuery("SELECT to_regclass('$schema.$table')")
         resultSet.next()
-        return resultSet.getString(1) == table
+        return resultSet.getString(1) == "$schema.$table"
```

Comment:
@mnonnenmacher, please check that this change actually works for you and does not cause table creation to go into an endless loop if your version of PostreSQL really only returns the table name without the scheme name.

Reply:
The related test cases are failing, maybe this is the reason.

Reply:
Fixed.